presto-main/src/main/java/com/facebook/presto/operator/PartitionedOutputOperator.java

```diff
@@ -298,14 +310,17 @@ public Page getOutput()
         private final List<Optional<Block>> partitionConstants;
         private final PagesSerde serde;
         private final List<PageBuilder> pageBuilders;
+        private final boolean replicateAny;
```

Comment:
judging just from the variable name, `replicateAny` would be true if we're supposed to replicate just nulls. Let's name it `replicateFirstRow`, and the enum value too.